src/Microsoft.Management.Configuration.Processor/Extensions/ValueSetExtensions.cs

```diff
@@ -175,5 +177,61 @@ public static bool ContentEquals(this ValueSet first, ValueSet second)
 
             return true;
         }
+
+        /// <summary>
+        /// Converts the value set to YAML like output.
+        /// </summary>
+        /// <param name="set">The set to output.</param>
+        /// <returns>The string.</returns>
+        public static string ToYaml(this ValueSet set)

```

Comment:
I saw this is only used in tests. Is it expected to be used in product code in the future? If not, shall we move the code to test common?

Reply:
I honestly didn't notice that this was in the processor; I just assumed that it was in the unit tests as that is how I navigated there (via F12 from a test).